### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#332] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-23 103945](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fcb782bd-0865-4ac3-8c2f-bd72aaedc7b0)

**Comment by Elia-Belli** [comment]
$$\mathcal{B}(M_{33}(\mathbb{R}))=\\{E_{ij}\in M_{33}(\mathbb{R})|\ e_{ij}=1\\}=\\{insieme\ delle\ matrici\ 3x3\ nulle\ con\ 1\ nella\ casella\ ij\\}$$

$$\mathcal{B}(\mathcal{S}_{33}(\mathbb{R}))=\begin{pmatrix}
1 & 0 & 0 \\
0 & 0 & 0 \\
0 & 0 & 0 
\end{pmatrix}\begin{pmatrix}
0 & 0 & 0 \\
0 & 1 & 0 \\
0 & 0 & 0 
\end{pmatrix}
\begin{pmatrix}
0 & 0 & 0 \\
0 & 0 & 0 \\
0 & 0 & 1 
\end{pmatrix}
\begin{pmatrix}
0 & 1 & 0 \\
1 & 0 & 0 \\
0 & 0 & 0 
\end{pmatrix}\begin{pmatrix}
0 & 0 & 1 \\
0 & 0 & 0 \\
1 & 0 & 0 
\end{pmatrix}
\begin{pmatrix}
0 & 0 & 0 \\
0 & 0 & 1 \\
0 & 1 & 0 
\end{pmatrix}$$

$$\mathcal{B}(\mathcal{A}_{33}(\mathbb{R}))=
\begin{pmatrix}
0 & -1 & 0 \\
1 & 0 & 0 \\
0 & 0 & 0 
\end{pmatrix}\begin{pmatrix}
0 & 0 & -1 \\
0 & 0 & 0 \\
1 & 0 & 0 
\end{pmatrix}
\begin{pmatrix}
0 & 0 & 0 \\
0 & 0 & -1 \\
0 & 1 & 0 
\end{pmatrix}$$

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b60821e1-b6bb-4e7c-a314-d2b711e40868)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/de3ff993-23ff-4b4f-81d2-45c42004a9b5)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b6d2fbbf-7913-4567-89f0-f973215fb281)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/6f1481f1-49f3-4759-83a3-52d426c8bba8)
